packages/rn-tester/js/examples/TextInput/TextInputSharedExamples.js

```diff
@@ -19,6 +19,7 @@ const {
   TextInput,
   View,
   StyleSheet,
+  Switch,
```

Comment:
Just noticed this. Technically all the diffs here need macOS diff tags too, otherwise they might get wiped in the next merge.